infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/StreamsSnapshotWriter.java

```diff
@@ -106,15 +104,12 @@ public StreamsSnapshotWriter(CorfuRuntime rt, LogReplicationConfig config, LogRe
     private void initializeShadowStreams(LogReplicationConfig config) {
         // For every stream create a shadow stream which name is unique based
         // on the original stream and a suffix.
-        for (String streamName : config.getStreamsToReplicate()) {
-            String shadowStreamName = streamName + SHADOW_STREAM_SUFFIX;
-            UUID streamId = CorfuRuntime.getStreamID(streamName);
+        for (UUID streamId : config.getStreamsInfo().getStreamIds()) {
+            String shadowStreamName = streamId + SHADOW_STREAM_SUFFIX;
```

Comment:
The shadow stream name will now use the stream id instead of name.  Is it intentional?